frontend/public/extend/devconsole/components/pipelines/PipelineVisualizationStepList.scss

```
@@ -0,0 +1,23 @@
+.odc-pipeline__steps {
```

This should be a block, not an element.
```suggestion
.odc-pipeline-vis-step-list {
```